lang/en.json

```diff
@@ -86,6 +86,9 @@
   "TAB.NOTE": "NOTE",
   "TAB.BUILDING": "BUILDING",
   "TAB.HIRELING": "HIRELING",
+  "TAB.ATTRIBUTES": "ATTRIBUTES",
+  "TAB.CRAFT_SUPPLY": "CRAFT & SUPPLY",
+  "TAB.ARTIFACT": "ARTIFACT",
```

Comment:
Cool! Here are Swedish translations:

```
  "TAB.CRAFT_SUPPLY": "TILLVERKNING",
  "TAB.ARTIFACT": "ARTEFAKT",
```

Or if your prefer a more direct translation of Craft & Supply:

```
  "TAB.CRAFT_SUPPLY": "HANTVERK & LAGER",
```